### PR TITLE
Vi 36

### PR DESCRIPTION
**Pull request by mSobonVirtua**

### CODE REVIEW

File: my-project/src/Controller/ProductCategoryController.php [hunk 1]
```diff
@@ -58,6 +70,7 @@ public function new(Request $request): Response
             }
             catch(\Exception $exception)
             {
```
Comment (astyczen) [inline my-project/src/Controller/ProductCategoryController.php:72]: @mSobonVirtua remove commented out line

File: my-project/src/Controller/ProductCategoryController.php [hunk 2]
```diff
@@ -145,6 +169,45 @@ public function delete(Request $request, ProductCategory $productCategory): Resp
...
+                $em->persist($image);
+                $em->flush();
+
+                $this->addFlash(
+                    'notice',
+                    'Your Image was added'
+                );
+            }
+            catch(\Exception $exception)
+            {
```
Comment (astyczen) [inline my-project/src/Controller/ProductCategoryController.php:201]: @mSobonVirtua remove commented out line

File: my-project/templates/product_category/addImages.html.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{#/**#}
+{#* VI-36#}
+{#*#}
+{#* @category   Virtua#}
```
Comment (astyczen) [inline my-project/templates/product_category/addImages.html.twig:4]: @mSobonVirtua update all headers with module name and description in category, like CategoryGallery

File: my-project/templates/product_category/_form.html.twig [hunk 4]
```diff
@@ -10,4 +10,4 @@
 {{ form_start(form) }}
     {{ form_widget(form) }}
     <button class="btn">{{ button_label|default('Save') }}</button>
-{{ form_end(form) }}
```
Comment (astyczen) [inline my-project/templates/product_category/_form.html.twig:13]: @mSobonVirtua do not commit such changes id not nessessary, better to rever the file before commit if inly such changes are made

File: my-project/src/Repository/ImageRepository.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+ * @method Image[]    findAll()
+ * @method Image[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ImageRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Image::class);
+    }
+
```
Comment (astyczen) [inline my-project/src/Repository/ImageRepository.php:21]: @mSobonVirtua remove unneeded code

File: my-project/src/Services/FileUploaderService.php [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        $this->imageMaxSize = 250000;
+    }
+
+    /**
+     * @param UploadedFile $file
+     * @return string
+     * @throws FileException
+     */
+    public function upload(UploadedFile $file)
+    {
```
Comment (astyczen) [inline my-project/src/Services/FileUploaderService.php:32]: @mSobonVirtua is this check needed? if not remove commented out code

File: my-project/templates/product_category/_edit_form.html.twig [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+<script>
+    const addImageForm = document.forms.namedItem('image');
+    addImageForm.addEventListener('submit', (e)=>{
+        e.preventDefault();
+        const dataForm = new FormData(addImageForm);
+        const response = fetch('{{ path('UploadAndAddImageToCategory', {'id': product_category.id}) }}', {
+            method: 'POST',
+            body: dataForm
+        });
```
Comment (astyczen) [inline my-project/templates/product_category/_edit_form.html.twig:33]: @mSobonVirtua remove console logs

File: my-project/templates/product_category/addImages.html.twig (hunk 3 above)
Comment (mSobonVirtua) [reply my-project/templates/product_category/addImages.html.twig:4]: @astyczen Something like that ?
{#/**#}
{#* VI-36 AddImage#}
{#*#}
{#* @category   Add image to category#}
{#* @package    AddImage_Module#}
{#* @copyright  Copyright (c) Virtua#}
{#* @author     Name <m.sobon@wearevirtua.com>#}
{#*/#}

Comment (astyczen) [reply my-project/templates/product_category/addImages.html.twig:4]: {#/**#}
{#* VI-36 AddCategoryImage#}
{##}
{# @category CategoryImages#}
{#* @Package Virtua_AddCategoryImage#}
{#* @copyright Copyright (c) Virtua#}
{#* @author Mateusz Sobon m.sobon@wearevirtua.com#}
{#*/#}

**Pull request closed without merging**